tests/test_qmakebuild/test1.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
```

Comment:
I will create a 'sanity' subset of proto messages to do not copy it each time later :)